validator/src/journal/candidate_block.rs

```diff
@@ -486,7 +471,31 @@ impl CandidateBlock {
                     .expect("BlockBuilder has no method 'build_block'"),
             ),
             pending_batches,
-        ))
+        )
+    }
+
+    fn consensus_not_finalize_block(&self, builder: &cpython::PyObject) -> bool {
```

Comment:
This function name is very confusing.

Reply:
Would `consensus_does_not_finalize_block` be better?

Reply:
That makes more sense.  Rename it like so

Reply:
This is a helper method in CandidateBlock. I find it very unfortunate that the coding style of the python consensus interface leaks all the way into this helper method in the Rust candidate block. Since I put `!` in the method shouldn't there be some indication to the programmer reading it that there is a `not` check?